utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -105,7 +105,7 @@ pub struct StorageParams {
 	/// Trie cache size in bytes.
 	///
 	/// Providing `0` will disable the cache.
-	#[clap(long, default_value = "1024")]
+	#[clap(long, default_value = "0")]
```

Comment:
Otherwise it would not get enabled when you set `enable_trie_cache`.

Reply:
> After thinking about this more. I'm not sure how it worked before?

I am not sure either. Maybe the state-cache just did not have such a large influence as the Trie cache now does.  
But yea, the former results should already be wrong due to state-caching. I did not notice this earlier, as `--state-cache-size` did not make a noticeable difference.  

> Otherwise it would not get enabled when you set enable_trie_cache.

I did not put a `--enable-trie-cache` for the `storage` command, since the `trie-cache-size` was not inherited from the import commands. But I will add it for consistency.

Reply:
Wait I think this is bytes, will change to `67108864`.

Reply:
> Well, we need to measure worst case performance… so a second iteration of a benchmark should not rely on a former iteration. None of the benches should actually rely on a cache.

Makes sense, but how do we then handle values which we assume of already being present? Aka the whitelisted values? Do we just somehow remove them from the formular that gives us the weight?

Reply:
> Do we just somehow remove them from the formular that gives us the weight?

For pallet weights: Yes.  
For storage weights: Does not matter, since we need the average read/write weight for a key, regardless of whitelist, since whitelisted values are also read once.  
For `BlockExecutionWeight` and `ExtrinsicBaseWeight`: No, since this is where we want to account for the cost of whitelisted keys, so they are included here.

https://github.com/paritytech/substrate/blob/7de3d7870f63a525977fd4e91cda16039e7b1d60/client/db/src/bench.rs#L524-L531

That could be written a bit nicer with a `.filter` :see_no_evil: